translations/messages.en.yaml

```diff
@@ -57,8 +57,10 @@ cover: Cover
 related_posts: Related posts
 random_posts: Random posts
 federated_magazine_info: This magazine is from a federated server and may be incomplete.
+disconnected_magazine_info: This magazine is not receiving updates (last activity %days% day(s) ago). Subscribe to start receiving updates.
+always_disconnected_magazine_info: This magazine has not yet received any updates. Subscribe to start receiving updates.
 federated_user_info: This profile is from a federated server and may be incomplete.
-go_to_original_instance: Browse more on the original instance.
+go_to_original_instance: For a complete list of posts, browse on the original instance.
```

Comment:
maybe: `For a complete list of posts, visit the original instance.`? I am not sure though

Reply:
```suggestion
go_to_original_instance: For a complete list of posts, visit the original instance.
```